PublicMapping/districtbuilder
# Style cleanup for organization page

## Overview

Cleans up formatting for the organization page.

### Demo

![Screen Shot 2021-03-26 at 6 21 21 PM](https://user-images.githubusercontent.com/1809908/112698586-58a2c900-8e60-11eb-9bc2-8da24b508c8a.png)

### Notes

- Added color the the featured maps

## Testing Instructions

- Go to an organization page
- Join and unjoin the organization
- Make sure the page looks like the screenshot and generally seems okay, doesn't appear to have any missing styles

Fixes #645
